src/processor.ts

```diff
@@ -54,18 +60,53 @@ processor.run(new TypeormDatabase(), async ctx => {
                             nonce,
                             result,
                             blockNumber,
-                            timestamp: new Date(timestamp)
+                            timestamp,
                         }));
                     }
 
                 }
             }
+            else if (item.kind === "event") {
+                if (WHITELIST_CONFIG.Events.includes(item.event.name)) {
+                    let recipient = item.event.args.who;
+                    // in bridge transfers, worker accounts can't be the final recipient of the index business
+                    // because we have no access to the private key of the business recipient
+                    // we can't emit a bridge transfer using the business recipient account.
+                    // we can only control workers
+                    // therefore it makes sense to filter out those whose are not in the whitelist
+                    if (!WHITELIST_CONFIG.Accounts.includes(recipient)) {
+                        continue;
+                    }
+                    let id = item.event.id;
+                    let name = item.event.name;
+                    // it might be large
+                    let amount = item.event.args.amount as string;
```

Comment:
So here we assume all event args have the field `amount`? Not sure if they are the same everywhere. (So far I haven't found one not to be though)